src/main/java/ro/utcn/daneeee/assignment1/model/Question.java

```diff
@@ -0,0 +1,88 @@
+package ro.utcn.daneeee.assignment1.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import sun.util.BuddhistCalendar;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    private Integer authorId;
+    private String title;
+    private String text;
+    @ToString.Exclude
+    private Date creationDate;
+    //private List<String> tags;
```

Comment:
Commented code -- in the future you should try to keep your code clean (and use git to keep older versions).

Also, the original intention of having the tags directly as a list inside the question was a clearly better design. Maybe we should discuss at the lab why you did not keep this original design.